tests/e2e/specs/admin-tracking.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+import { URL } from 'url';
+import { resetSiteKit } from '../utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeAll( async() => {
+		await activatePlugin( 'google-site-kit' );
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await resetSiteKit();
+	} );
+
+	beforeEach( async() => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
+		await page.waitForSelector( '.mdc-tab-scroller__scroll-content' );
+		page.click( '.mdc-tab-scroller__scroll-content button:last-child' );
+	} );
+
+	afterAll( async() => {
+		await resetSiteKit();
+		await deactivatePlugin( 'google-site-kit' );
+		await deactivatePlugin( 'e2e-tests-auth-plugin' );
+	} );
+
+	it( 'Should select opt-in by default', async() => {
```

Comment:
We currently begin the test names with lowercase letters, let's do this here as well for consistency reasons.

Reply:
All test names update to lower case.